README.md

```diff
@@ -24,15 +23,14 @@ See [Stockpile API](https://github.com/AdamVig/stockpile-api) for the Restful AP
 **Note:** Some native functionality like scanning won't be available in the browser. You have to run the app on an emulator or device for the full experience.
 
 ## Running on a Device or Emulator
-Running apps on a device or emulator requires you to build the app on your computer. To do this, you have to set up the Android SDK and Apple's Xcode. For detailed instructions on how to do this, follow [this guide for Mac](https://ionicframework.com/docs/v2/resources/platform-setup/mac-setup.html) or [this one for Windows](https://ionicframework.com/docs/v2/resources/platform-setup/windows-setup.html).
+Running apps on a device or emulator requires you to build the app locally. To do this, you have to set up the Android SDK and/or Apple's Xcode. For detailed instructions on how to do this, follow [this guide for Mac](https://ionicframework.com/docs/v2/resources/platform-setup/mac-setup.html) or [this one for Windows](https://ionicframework.com/docs/v2/resources/platform-setup/windows-setup.html).
 
-- Run `ionic state restore` to prepare the iOS and Android platforms and install the correct plugins for each
-- Run `ionic emulate ios` or `ionic emulate android` to emulate
-- Run `ionic run ios` or `ionic run android` to run on real device
+- Add platforms you want to run with `ionic cordova platform add ios` or `ionic cordova platform add android`
+- Run `ionic cordova prepare` to install the plugins for each platforms
+- Run `ionic cordova  emulate ios` or `ionic cordova emulate android` to emulate
+- Run `ionic cordova run ios` or `ionic cordova run android` to run on real device
```

Comment:
> to run on real device
I would add "...connected to your computer," unless you think that's too obvious.